import.go

```diff
@@ -133,19 +133,28 @@ func (i *Importer) Add(exportNode *ExportNode) error {
 	if node.subtreeHeight == 0 {
 		node.size = 1
 	} else if stackSize >= 2 && i.stack[stackSize-1].subtreeHeight < node.subtreeHeight && i.stack[stackSize-2].subtreeHeight < node.subtreeHeight {
-		node.leftNode = i.stack[stackSize-2]
-		node.rightNode = i.stack[stackSize-1]
-		node.leftNodeKey = node.leftNode.nodeKey
-		node.rightNodeKey = node.rightNode.nodeKey
-		node.size = node.leftNode.size + node.rightNode.size
+		leftNode := i.stack[stackSize-2]
+		rightNode := i.stack[stackSize-1]
+
+		node.leftNode = leftNode
+		node.rightNode = rightNode
+		node.leftNodeKey = leftNode.nodeKey
+		node.rightNodeKey = rightNode.nodeKey
+		node.size = leftNode.size + rightNode.size
```

Comment:
It seems to me that you moved leftNode and rightNode to vars and re-used, but isn't that the same has `node.leftNode` and `node.rightNode` or does GO have an issue with using the object?

Reply:
Yes, it's equivalent, it's just I think it makes code shorter this way, because the node.leftNode will be repeated many times